contrib/custom_top_level_templates.go

```diff
@@ -113,10 +117,14 @@ const (
 // Returns the template for generating input snapshots.
 func InputSnapshotManualBuilder(params SnapshotTemplateParameters) model.CustomTemplates {
 	templateContentsBytes, err := ioutil.ReadFile(templatesDir + InputSnapshotManualBuilderCustomTemplatePath)
+	var templateContents string
 	if err != nil {
-		panic(err)
+		if err != nil {
```

Comment:
I think this duplicate error check is unnecessary

Reply:
Same comment as above